libraries/rush-lib/assets/rush-init/rush.json

```diff
@@ -426,7 +426,7 @@
        * locally linked because it would create a cyclic dependency; instead, the last published
        * version will be installed in the Common folder.
        */
```

Comment:
Let's update the docs:
```js
      /**
       * A list of Rush project names that are allowed to be installed from NPM
       * instead of locally linking the folder.
       *
       * If a project's package.json specifies a dependency that is another Rush project
       * in the monorepo workspace, normally Rush will locally link its folder instead of
       * installing from NPM.  If you are using PNPM workspaces, this is indicated by
       * a SemVer range such as "workspace:^1.2.3".  To prevent mistakes, Rush reports
       * an error if the "workspace:" protocol is missing.
       *
       * Locally linking ensures that regressions are caught as early as possible and is
       * a key benefit of monorepos.  However there are occasional situations where
       * installing from NPM is needed.  A classic example is a cyclic dependency.
       * Imagine three Rush projects: "my-toolchain" depends on "my-tester", which depends
       * on "my-library".  Suppose that we add "my-toolchain" to the "devDependencies"
       * of "my-library" so it can be built by our toolchain.  This cycle creates 
       * a problem -- Rush can't build a project using a not-yet-built dependency.  
       * We can solve it by adding "my-toolchain" to the "decoupledLocalDependencies"
       * of "my-library", so it builds using the last published release.  Choose carefully
       * which package to decouple; some choices are much easier to manage than others.
       *
       * (In older Rush releases, this setting was called "cyclicDependencyProjects".)
       */
```

Reply:
updated 👍